draft-ietf-quic-transport.md

```diff
@@ -1128,12 +1129,16 @@ initial_max_data (0x0001):
   sending a MAX_DATA ({{frame-max-data}}) for the connection immediately after
   completing the handshake.
 
-initial_max_stream_id (0x0002):
+initial_max_stream_id_bidi (0x0002):
 
 : The initial maximum stream ID parameter contains the initial maximum stream
-  number the peer may initiate, encoded as an unsigned 32-bit integer.  This is
-  equivalent to sending a MAX_STREAM_ID ({{frame-max-stream-id}}) immediately
-  after completing the handshake.
+  number the peer may initiate for bidirectional streams, encoded as an unsigned
+  32-bit integer.  This value MUST be a valid bidirectional stream ID for a
+  peer-initiated stream (that is, a server includes a value that have the least
```

Comment:
/s/have/has

Reply:
I think the phrasing below is a bit cleaner, so I'd copy that.  "that is, the two least significant bits are set to 0 by a server and to 1 by a client"